src/aselo-webchat.tsx

```diff
@@ -138,7 +140,15 @@ export const initWebchat = async () => {
 
   const changeLanguageWebChat = getChangeLanguageWebChat(manager, currentConfig);
 
-  changeLanguageWebChat(initialLanguage);
+  const setChangeLanguageWebChat = (external: string | null | undefined, language: string) => {
```

Comment:
Would it not be simpler to do:
```
changeLanguageWebChat(externalWebChatLanguage ?? initialLanguage);
```
rather than defining the `setChangeLanguageWebChat` function?